LaunchAPPL/Client/MiniVMac.cc

```diff
@@ -423,7 +477,8 @@ bool MiniVMac::CheckOptions(variables_map &options)
         && options.count("minivmac-dir") != 0
         && options.count("minivmac-rom") != 0
         && options.count("system-image") != 0
-        && (options.count("autoquit-image") + options.count("autquit7-image")) == 1;
+        && options.count("autoquit-image") != 0
```

Comment:
I'd say `count(autoquit) + count(autquit7) >= 1` here so that System 6 only users don't have to change anything.
If the wrong one is specified, we'll get an error message anyway.

Reply:
@autc04 I didn't fix this, sorry, I am a little busy in the next couple of days, but I'm happy to address this afterwards. Thanks for merging anyway, this is an amazing project 🙂

Reply:
No problem, I thought as much. I don't want to frustrate you by drawing things out endlessly with minor nitpicks, so when I happened to have some free time myself I merged and made this change myself (in the master branch).

Of course, there's still plenty of stuff to do. Better error handling, various ways of handling MPW Tools, and a way of handling applications that need supporting files or folders installed next to themselves. And that's for LaunchAPPL, and I did not have to think hard to come up with that list. So I'm always looking forward to future contributions :smile: